src/main/java/com/google/sps/servlets/VotingServlet.java

```diff
@@ -51,16 +51,26 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     PreparedQuery results = datastore.prepare(query);
 
     // Checks if a review exists, and if the current user has already voted on it. 
-    if(review != null && results.countEntities() < 1) {
+    if(review != null) {
+
+      String vote = "negative";
+      if (!upvotes.equals(review.getProperty("positive"))){
+        vote = "positive";
+      }
```

Comment:
String vote = upvotes.getProperty("positive") ? "negative" : "positive";

Reply:
I actually have never seen this code style and don't fully understand what this line means.